### PR TITLE
dlgInfill

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

**Comment by lilyclements** [comment]
Is this ready for review?

**Comment by stevenndungu** [comment]
Yes, it is.

**Comment by stevenndungu** [comment]
@africanmathsinitiative/developers this is ready for re-review. Required changes were made.

**Comment by lilyclements** [comment]
@dannyparsons This is ready for you to review

File: instat/dlgInfill.vb [hunk 1]
```diff
@@ -52,12 +55,18 @@ Public Class dlgInfill
...
+        ucrBase.clsRsyntax.SetBaseRFunction(clsDefaultFunction.Clone())
+
     End Sub
+
+    Private Sub SetRCodeforControls(bReset As Boolean)
+        SetRCode(Me, ucrBase.clsRsyntax.clsBaseFunction, bReset)
+    End Sub
+
     Private Sub TestOkEnabled()
         If Not (ucrReceiverDate.IsEmpty) OrElse Not (ucrReceiverFactors.IsEmpty) Then
```
Comment (dannyparsons) [inline instat/dlgInfill.vb]: Is this the right test? Surely it's just the Date receiver that is needed.

Comment (stevenndungu) [inline instat/dlgInfill.vb]: Ok, adjusting that shortly

**Comment by stevenndungu** [comment]
change made

**Pull request merged**